### PR TITLE
travis: test on Windows

### PR DESCRIPTION
**Pull request by targos**
Fixes: https://github.com/nodejs/node-core-utils/issues/313

### CODE REVIEW

**Comment by targos** [comment]
Let's see if it works

**Comment by joyeecheung** [comment]
I think adding these to `.gitattributes` should fix the Windows issue?

```
# auto detect text files
* text=auto

# we don't have actual binary files in this repo
*.* text eol=lf
```

**Comment by targos** [comment]
I changed the matrix and created `.gitattributes`: https://github.com/nodejs/node-core-utils/pull/314/checks

**Comment by targos** [comment]
I cannot reproduce on my Windows machine

**Comment by targos** [comment]
Rebased. @joyeecheung do you think we still need the `.gitattributes`?

**Comment by targos** [comment]
There seems to still be issues with CRLF

**Comment by joyeecheung** [comment]
Can you rebase and force push to see if Travis can pick the other fix up?

**Comment by joyeecheung** [comment]
@targos ah, sorry, didn't see the history (was on mobile)

**Comment by joyeecheung** [comment]
The previous green Windows I got is on travis-ci.org instead of travis-ci.com, I wonder if that makes a difference... https://travis-ci.org/joyeecheung/node-core-utils/jobs/462560512

Also, looks like the failing tests are all comparing output to some files contents

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Auto-detect text files
+* text=auto
```
Comment (joyeecheung) [inline .gitattributes]: Can you try simply using

```
* text=auto eol=lf
```

in the whole file?

**Pull request closed without merging**